### DISCUSSION TITLE
It doesn't boot after installation

### DISCUSSION THREAD

**Discussion by ctwhome** [AdityaGarg8/mbp-16.x-ubuntu-kernel#23] [Q&A] [answered]
Hi! Thanks for this incredible effort.
I have installed Ubuntu on a MacBook pro 2019,the live goes well and the installation occurs without problems, but after restarting and selecting the Ubuntu drive the computer stays black, it doesn't do anything.

I didn't create different partitions for efi, swap... Andnso on, that might be the problem?

Thanks!

**Comment by AdityaGarg8** [comment, marked as answer]
You may need to use refind

https://wiki.t2linux.org/guides/refind/

**Reply by ctwhome** [reply]
Im going to try thank you

**Reply by ctwhome** [reply]
It worked! Thanks!
